pwiz_tools/Skyline/Controls/Graphs/GraphFullScan.cs

```diff
@@ -540,6 +518,115 @@ private void CreateIonMobilityHeatmap()
             heatMapGraphPane.SetPoints(_heatMapData, minDrift, maxDrift);
         }
 
+        private void PopulateProperties()
+        {
+            bool hasIonMobilityDimension = _msDataFileScanHelper.MsDataSpectra.Length > 1 ||
+                                           _msDataFileScanHelper.MsDataSpectra.First().IonMobilities != null;
+
+            var spectra = _msDataFileScanHelper.MsDataSpectra;
+            FullScanProperties spectrumProperties = null;
+            if (spectra.Any())
+            {
+                spectrumProperties = FullScanProperties.CreateProperties(spectra[0]);
+
+                var ces = _msDataFileScanHelper.MsDataSpectra.SelectMany(spectrum => spectrum.Precursors)
+                    .Select(pre=> pre.PrecursorCollisionEnergy).Where(ce => ce.HasValue).Select(ce => ce.Value)
+                    .Distinct().ToArray();
+                if (ces.Length == 1)
+                    spectrumProperties.CE = ces[0].ToString(Formats.OPT_PARAMETER);
+
+                var precursor = _msDataFileScanHelper.ScanProvider.Transitions
+                    .FirstOrDefault(t => (t.Id as Transition)?.IonType == IonType.precursor);
+                if (precursor != null)
+                {
+                    spectrumProperties.PrecursorMz = precursor.PrecursorMz.Value.ToString(Formats.Mz);

```

Comment:
`FullScanProperties.CreateProperties(spectra[0])` already set the precursor m/z based on the precursor in the spectrum.
It looks like this line of code is replacing that precursor value with the precursor of something in the document.
I don't think that's what users will want: they want to know what the mass spectrometer said the precursor was for the spectrum that they are looking at.
(same with collision energy and charge state)

Reply:
This is a "philosophical" question. Since the full scan viewer is not synced with the targets tree the user needs to know for what precursor in the document this spectrum was recorded. Maybe both should be shown, one in the Acquisition and the other on the Precursor sections.